shotover-proxy/tests/runner/hotreload_int_tests.rs

```diff
@@ -1,10 +1,81 @@
 use crate::shotover_process;
+use redis::{Client, Commands};
+use std::time::Duration;
+use test_helpers::docker_compose::docker_compose;
+use tokio::time::sleep;
+
 #[tokio::test]
-async fn test_hotreload_cli_flag_accepted() {
-    let shotover_process = shotover_process("tests/test-configs/null-valkey/topology.yaml")
+async fn test_hotreload_basic_redis_connection() {
+    let _compose = docker_compose("tests/transforms/docker-compose-moto.yaml");
+    sleep(Duration::from_millis(2000)).await;
```

Comment:
docker_compose and shotover_process contain their own logic for waiting until shotover or docker have finished starting up. Can you try removing these sleeps and seeing if it still works fine?